tests/modules/vcf2maf/vep/homo_sapiens/106_GRCh38/chr_synonyms.txt

```diff
@@ -0,0 +1,57 @@
+KI270928.1	chr22_KI270928v1_alt
```

Comment:
Test data should ideally go to the nf-core/test-datasets repo and there into the modules branch (ideally based on already existing test data). Is there a reason this file can't be added there? :)

Reply:
Ah yes I have run into this before. I just ran the UNTAR module then before my actual module. see here: https://github.com/nf-core/modules/blob/master/tests/modules/controlfreec/freec/main.nf depends of course on how your process expects these files

Reply:
😆